src/test/java/org/mule/extension/db/integration/select/SelectTestCase.java

```diff
@@ -83,6 +83,11 @@ public void maxRows() throws Exception {
   }
 
   @Test
+  public void emptyResult() throws Exception {
+    flowRunner("emptyResult").run().getMessage();
+  }
```

Comment:
@pablomusumeci Can we add any other assertions here?

Reply:
you can take the returned iterator and verify that hasNext() is false